### PR TITLE
Mark headings in XLIFFs so that ATE can indicate that these fields have SEO significance.

### PR DESCRIPTION
**Pull request by kagg-design**
https://onthegosystems.myjetbrains.com/youtrack/issue/wpmltm-3081

### CODE REVIEW

**Comment by kagg-design** [comment]
@OnTheGoSystems/wpml Please review only first commit, as second one contains just code formatting according to WPCS.

**Review by strategio — DISMISSED** [review]
+1 @kagg-design

File: src/class-wpml-beaver-builder-translatable-nodes.php [hunk 1]
```diff
@@ -68,16 +88,23 @@ public function update( $node_id, $settings, WPML_PB_String $string ) {
...
+						 *
+						 * @var WPML_Beaver_Builder_Module_With_Items $node
+						 */
 						$node = new $node_data['integration-class']();
 						$node->update( $node_id, $settings, $string );
+						// phpcs:disable Generic.CodeAnalysis.EmptyStatement.DetectedCatch
 					} catch ( Exception $e ) {
-
+						// Nothing to do with the exception, we do not handle it.
+						// phpcs:enable
```
Comment (andreasciamanna) [inline src/class-wpml-beaver-builder-translatable-nodes.php]: @kagg-design shouldn't be `// phpcs:enable Generic.CodeAnalysis.EmptyStatement.DetectedCatch`?

File: src/class-wpml-beaver-builder-translatable-nodes.php [hunk 2]
```diff
@@ -98,15 +127,35 @@ public function get_string_name( $node_id, $field, $settings ) {
...
+	 * Get wrap tag for string.
+	 * Used for SEO, can contain (h1...h6, etc.)
+	 *
+	 * @param stdClass $settings Field settings.
+	 *
+	 * @return string
+	 */
+	private function get_wrap_tag( $settings ) {
+		if ( isset( $settings->type ) && 'heading' === $settings->type ) {
+			if ( isset( $settings->tag ) ) {
```
Comment (andreasciamanna) [inline src/class-wpml-beaver-builder-translatable-nodes.php]: You can merge these two conditions into one

**Review by andreasciamanna — CHANGES_REQUESTED** [review]

File: src/class-wpml-beaver-builder-translatable-nodes.php (hunk 1 above)
Comment (kagg-design) [reply src/class-wpml-beaver-builder-translatable-nodes.php]: @andreasciamanna it is on line 104, before `catch (`

File: src/class-wpml-beaver-builder-translatable-nodes.php (hunk 2 above)
Comment (kagg-design) [reply src/class-wpml-beaver-builder-translatable-nodes.php]: @andreasciamanna done.

File: src/class-wpml-beaver-builder-translatable-nodes.php (hunk 1 above)
Comment (andreasciamanna) [reply src/class-wpml-beaver-builder-translatable-nodes.php]: @kagg-design please don't close a conversation with no valid reason.

I can obviously see what's in line 104, but line 104 is for **disabling** the rule.  
The line I'm commenting is for **enabling** a rule.  
When you disable a rule, you should re-enable it by specifying the name of that rule.

If you disable 3 rules, how phpcs can know which one you are re-enabling?

Comment (kagg-design) [reply src/class-wpml-beaver-builder-translatable-nodes.php]: @andreasciamanna Unfortunately, I didn't understand you at first time. Now I have made required changes.

**Review by andreasciamanna — DISMISSED** [review]

**Pull request merged**